corehq/feature_previews.py

```diff
@@ -127,3 +127,12 @@ def enable_callcenter(domain_name, checked):
     ),
     privilege=LOOKUP_TABLES,
 )
+
+
+SHARE_CASE_TILE_CONTEXT = FeaturePreview(
```

Comment:
also `SHOW_PERSIST_CASE_CONTEXT_SETTING` seems like being used only by a couple of active real domains and a bunch of test domains so this sounds safe enough to move under `SHOW_PERSIST_CASE_CONTEXT_SETTING` only.